toolversions.sh

```diff
@@ -42,6 +42,7 @@ install_docfx() {
   if [[ ! -f $DOCFX ]]
   then
     (echo "Fetching docfx v${DOCFX_VERSION}";
+     mkdir -p $REPO_ROOT/packages;
```

Comment:
I've never tried using the nuget package instead of the zip file. I'll look into it - I can imagine reasons why it might not work, if it's more intended for cases where you're using the code in docfx. But if we *can* use it, we should :)